docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+# Setting up a pipeline for images
+
+## Objectives
+
+* Get familiarity on how to use multiple cloud services together.
+* Get familiarity on how to use cloud services using their command-line interfaces.
+
+The tutorial focuses on the following services:
+
+* Allas
+* cPouta
+* Pukki
+
+## Introduction
+
+We want to set up a simple pipeline which transforms the images that are given in input to it.
+
+![image-pipeline-diagram](../../img/image-pipeline-diagram.png)
+
+We use Allas as the place where to upload our images in input.
```

Comment:
```suggestion
We use Allas as the place to upload our images ( i.e. input).
```
I am not married to my sentence, but your sounds redundant.

Reply:
Text reorganized in https://github.com/CSCfi/csc-user-guide/pull/2424/commits/639462dbd338c7cc1a095593ed66cf5ab22ef700